### ISSUE TITLE
Is this a client or server side mod?

### ISSUE THREAD

**Issue by User4716** [JR1811/NeMuelch-hide-body-parts#3]

**Comment by JR1811** [comment]
The rendering is done on the client side.

_**I have not tested that yet**_, but if a "bare-bones" fabric client without that mod connects to a server, which has this mod installed, it shouldn't be much of a problem.

The entity would not be hidden on that specific client, but it should not be a game crashing issue. The player entity just has a nbt value that this client doesn't care about there.

All the entity needs, is to have the nbt values to let the client know, which body parts to render, if the client has that mod installed.

So basically, if you don't have it on the client side, you don't get the adjusted rendering and if you don't have it on the server side, it might still work with some workarounds but the easy nbt handling is done with this mod too.

**Comment by User4716** [comment]
Ok, thanks! I asked because I wasn't sure that was this is a vanilla feature players can modify without OP, or a custom thing that you coded, and only people who have this mod installed could see the modifications in your body
